includes/class-wc-payments.php

```diff
@@ -418,7 +419,20 @@ public static function add_plugin_links( $links ) {
 	 */
 	public static function register_gateway( $gateways ) {
 		$gateways[] = self::$card_gateway;
-		$gateways[] = self::$sepa_gateway;
+
+		if ( false === WC_Payments_Features::is_grouped_settings_enabled() ) {
```

Comment:
Don't make any changes to the gateways, unless the "grouped settings" feature is enabled (use WCPay Dev tool for it)

Reply:
This seems a little counter intuitive. We already have[ `_wcpay_feature_sepa` ](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payment-gateway-wcpay.php#L269) option and the [`_wcpay_feature_giropay`](https://github.com/Automattic/woocommerce-payments/pull/1577/files#diff-bcfd303d507c33a27671935e724ee64d0df30ff2a18bb6ef4c94f28d48190124R191) option. So to use SEPA we'd need to enable two options _wcpay_feature_sepa and _wcpay_feature_grouped_settings which will probably confuse members of the payments squads because things will stop working once this PR is merged. I think we should these feature flags mostly to show/hide things on the frontend and keep the backend free from flag checks that would drastically change it's behavior once we enabled the feature for everyone.